addons/website_mass_mailing/i18n/website_mass_mailing.pot

```diff
@@ -205,6 +205,12 @@ msgid ""
 "interesting. You can unsubscribe at any time."
 msgstr ""
 
+#. module: website_mass_mailing
+#: code:addons/website_mass_mailing/controllers/website_form.py:0
+#, python-format
+msgid "Mailing List(s) not found!"
+msgstr ""
```

Comment:
Seems strange that "M" is between "W" and "Y" :wink: This .pot file should be updated by exporting the translations via Odoo.

Reply:
Indeed, already requested that change: https://github.com/odoo/odoo/pull/113378#discussion_r1124669254
Make sure to ask if you are unsure of what we mean :)

Reply:
> Not sure if QSM found something to say, waiting before r+

Despite this :point_up:, no :wink: (although not tested), I let you handle this PR :wink: